[Python] Updated celery instrumentation documentation (#11600)
getsentry/sentry-docs: docs/platforms/python/integrations/celery/crons.mdx

Use the Celery integration to monitor your [Celery periodic tasks](https://docs.celeryq.dev/en/stable/userguide/periodic-tasks.html) and get notified when a task is missed (or doesn't start when expected), if it fails due to a problem in the runtime (such as an error), or if it fails by exceeding its maximum runtime.

<Note>

>   Please note that monitors will only be created on the task's first run.

Reviewer: ```suggestion
  Please note that a cron monitor will only be created the first time your task runs.
```

    },
}
```

<Note>
  Please note that only crontab parseable schedules will be successfully
  upserted.
</Note>

Next, we need to initialize Sentry. Where to do this depends on how you run beat:

- If beat is running in your worker process (that is, you're running your worker with the `-B`/`--beat` option), initialize Sentry in either the `celeryd_init` or `beat_init` signal.
- If beat is running in a separate process, you need to initialize Sentry in _both_ the `celeryd_init` and `beat_init` signal.

Make sure to also set `monitor_beat_tasks=True` in `CeleryIntegration`.

In addition to capturing errors, you can monitor interactions between multiple services or applications by [enabling tracing](/concepts/key-terms/tracing/). You can also collect and analyze performance profiles from real users with [profiling](/product/explore/profiling/).

Select which Sentry features you'd like to install in addition to Error Monitoring to get the corresponding installation and configuration instructions below.

<OnboardingOptionButtons
  options={["error-monitoring", "performance", "profiling"]}
/>

> ```python {filename:tasks.py} {"onboardingOptions": {"performance": "12-14", "profiling": "15-18"}}

Author: Good idea, added!
